subsys/bluetooth/mesh/net.c

```diff
@@ -661,18 +662,17 @@ static void bt_mesh_net_relay(struct net_buf_simple *sbuf,
 		goto done;
 	}
 
-	/* Sending to the GATT bearer should only happen if GATT Proxy
-	 * is enabled.
+	/* When the Friend node relays message for lpn, the message will be
+	 * retransmitted using the managed flooding security credentials and
+	 * the Network PDU shall be retransmitted to all network interfaces.
 	 */
 	if (IS_ENABLED(CONFIG_BT_MESH_GATT_PROXY) &&
-	    bt_mesh_gatt_proxy_get() == BT_MESH_GATT_PROXY_ENABLED) {
-		if (bt_mesh_proxy_relay(&buf->b, rx->ctx.recv_dst) &&
-		    BT_MESH_ADDR_IS_UNICAST(rx->ctx.recv_dst)) {
-			goto done;
-		}
+	    (rx->friend_match ||
+	     bt_mesh_gatt_proxy_get() == BT_MESH_GATT_PROXY_ENABLED)) {
+		bt_mesh_proxy_relay(&buf->b, rx->ctx.recv_dst);
```

Comment:
I don't think the destination can be both an LPN and a proxy client at the same time, can it? I.e. this looks like an unnecessary check here.

Reply:
Sorry, obviously it should be friend_cred.